docs/quickstart/20_setup_flowehr_locally.qmd

```diff
@@ -23,7 +65,29 @@ CI/CD instructions
 ::: {.panel-tabset}
 ### Locally
 
-Local instructions
+
+Before running the Makefiles in each repo:
+- `az login --tenant <Tenant ID>` which you can [find](https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/how-to-find-tenant) on the Azure Portal
+- `az account set --subscription <Subscription Name>`
+- `az account show` to check in correct subscription
+
+1. [FlowEHR-Data-Sources](https://github.com/UCLH-Foundry/FlowEHR-Data-Sources):
+- `make deploy` - keep note of the printing of connection details for use later 
+- One resource group is created in this step.
+- Error fixes:
+    - A common error is "ERROR: deleting subnet..." which seems to occur because a subnet already exists that is using the same address space so shouldn't be deleted. Therefore, change the address space
+
+2. [UCLH-FlowEHR](https://github.com/UCLH-Foundry/UCLH-FlowEHR):
+- `make all` - To deploy all infrastructure, and any configured pipelines and apps
+- `make infrastructure` - Alternatively, if you just want to deploy the infrastructure
+- Two resource groups are created in this step.
+- Error fixes:
+  - Re-run `make all`
+  - Manually delete resource group on Azure Portal
+    - In keyvaults - manage deleted keys and purge keyvault
+    - [ ] `make clean` should be updated (delete .tfstate terraform)
+    - Re-run `make all` fresh
```

Comment:
These should probably go into the public repos `dev_on_boarding.md` so that we can add more parts to these without having to track it in multiple locations